subgraphs/compound-v2/src/common/addresses.ts

```diff
@@ -0,0 +1,315 @@
+// store common addresses
+
+import { Address } from "@graphprotocol/graph-ts"
+
+// null address
+export const ADDRESS_ZERO = Address.fromString("0x0000000000000000000000000000000000000000")
+
+// "factory" address (Comptroller.sol)
+export const COMPTROLLER_ADDRESS = Address.fromString("0x3d9819210a31b4961b30ef54be2aed79b9c9cd3b")
+
+// PriceOracle Addresses
+export const PRICE_ORACLE1_ADDRESS = Address.fromString("0x02557a5E05DeFeFFD4cAe6D83eA3d173B272c904")
+
+// cToken addresses
+export const CAAVE_ADDRESS  =   Address.fromString("0xe65cdb6479bac1e22340e4e755fae7e509ecd06c")
```

Comment:
You shouldn't hardcode these. Otherwise we'd need to update the code and redeploy every time Compound adds a new market.

Reply:
fixed with data source template